Makefile

```diff
@@ -2,15 +2,21 @@ BUILD_FILES = $(shell go list -f '{{range .GoFiles}}{{$$.Dir}}/{{.}}\
 {{end}}' ./...)
 
 GH_VERSION ?= $(shell git describe --tags 2>/dev/null || git rev-parse --short HEAD)
+DATE_FMT = +%Y-%m-%d
+ifdef SOURCE_DATE_EPOCH
+    BUILD_DATE ?= $(shell date -u -d "@$(SOURCE_DATE_EPOCH)" "$(DATE_FMT)" 2>/dev/null || date -u -r "$(SOURCE_DATE_EPOCH)" "$(DATE_FMT)" 2>/dev/null || date -u "$(DATE_FMT)")
```

Comment:
Why the final ` date -u "$(DATE_FMT)"` fallback, which doesn't seem to respect SOURCE_DATE_EPOCH?

Reply:
It's taken verbatim from the reproducible builds efforts; https://reproducible-builds.org/docs/source-date-epoch/

My guess it's a guard regarding a future `date` regressions. I can go ask the community for a better detail or dig up some change logs if that desirable.

Reply:
Thanks for looking that up!